src/v2/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarExtraLinks.tsx

```diff
@@ -85,6 +93,11 @@ class ArtworkSidebarExtraLinksContainer extends React.Component<
     type: Schema.Type.Link,
   }))
   onClickBuyNowAskSpecialist() {
+    if (sd.ENABLE_V3_INQUIRY) {
+      this.props.showInquiry({ askSpecialist: true })
+      return
+    }
+
     this.props.mediator &&
       this.props.mediator.trigger &&
       this.props.mediator.trigger("openBuyNowAskSpecialistModal", {
```

Comment:
There's no actual difference between `onClickBuyNowAskSpecialist` and `onClickAuctionAskSpecialist`